services/src/main/java/org/keycloak/services/ServicesLogger.java

```diff
@@ -467,4 +467,8 @@ public interface ServicesLogger extends BasicLogger {
     @Message(id=106, value="Created script engine '%s', version '%s' for the mime type '%s'")
     @Once
     void scriptEngineCreated(String engineName, String engineVersion, String mimeType);
+
+    @LogMessage(level = DEBUG)
```

Comment:
I'd make it `INFO` just as a gentle reminder to a user that they are trying to create an admin user again.

Reply:
@pedroigor Does `INFO` work for you?

Reply:
I'm not sure. Do we really need to remind a user about the fact we did nothing? :) IMHO, it's sufficient to inform the user only about the successful creation of the admin and then about failed ops. I'd keep the `DEBUG` here. @vmuzikar WDYT?

Reply:
@mabartos Ok, fair enough. :) Let's keep `DEBUG`. We display `INFO` if admin is successfully created.